scripts/validate-packages.py

```diff
@@ -57,6 +57,11 @@ def _validate_revision(given_package, revision, path):
 
     packaging_version = package_json.get("packagingVersion", "2.0")
 
+    # validate upgrades version
+    min_dcos_release_version = package_json.get("minDcosReleaseVersion")
+    if packaging_version == "4.0" and LooseVersion(min_dcos_release_version) < LooseVersion("1.10"):
+        sys.exit("\tERROR\n\nminDcosReleaseVersion must be greater than or equal to 1.10")
```

Comment:
Absolutely, I made them optional. If upgradesFrom, and downgradesTo are null, then they are equivalent to the empty list.